blueocean-dist/pom.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this module really only exists for docker - perhaps a better name than "dist"? Or will it have other uses?